### PR TITLE
Lps 137699 additional tests

### PR DESCRIPTION
**Pull request by SamZiemer**
Hey @achaparro,

Here are 2 additional tests that test the logging of events in the upgrade report.

I was having a bit of an issue with testing error logging, due to the [LogAssertionTestRule](https://github.com/liferay/liferay-portal/blob/master/portal-test/src/com/liferay/portal/test/rule/LogAssertionTestRule.java#L316-L329) catching any error level log event I throw and fail the test. Let me know what you think we should do in this case.

### CODE REVIEW

**Comment by SamZiemer** [comment]
ci:test:sf

**Comment by SamZiemer** [comment]
ci:test:relevant

File: modules/apps/portal/portal-upgrade-impl/src/main/java/com/liferay/portal/upgrade/internal/report/UpgradeReport.java [hunk 1]
```diff
@@ -121,6 +130,112 @@ private String _getDialectInfo() {
...
+
+			String docLibPath =
+				PropsValues.LIFERAY_HOME + "/data/document_library";
+
+			if (_rootDir != null) {
+				docLibPath = _rootDir + "/data/document_library";
+			}
+
+			try {
+				Files.walkFileTree(
```
Comment (achaparro) [inline modules/apps/portal/portal-upgrade-impl/src/main/java/com/liferay/portal/upgrade/internal/report/UpgradeReport.java:176]: @SamZiemer, have you tried the performance of this? Do we have an estimation about how long it takes for a large number of files?

To simplify the code, do you think FileUtils.sizeOfDirectory() can be an option? We already use commons-io in portal-impl

File: modules/apps/portal/portal-upgrade-impl/src/main/java/com/liferay/portal/upgrade/internal/report/UpgradeReport.java [hunk 2]
```diff
@@ -121,6 +130,112 @@ private String _getDialectInfo() {
...
+					"the document library", StringPool.NEW_LINE);
+			}
+
+			final AtomicLong length = new AtomicLong(0);
+
+			String docLibPath =
+				PropsValues.LIFERAY_HOME + "/data/document_library";
+
+			if (_rootDir != null) {
+				docLibPath = _rootDir + "/data/document_library";
```
Comment (achaparro) [inline modules/apps/portal/portal-upgrade-impl/src/main/java/com/liferay/portal/upgrade/internal/report/UpgradeReport.java:172]: is this necessary? I mean, when we set the rootDir via configAdmin I believe (not totally sure) that that's the final folder and adding _/data/document_library_ is not needed.

**Review by achaparro — COMMENTED** [review]
Hi @SamZiemer,

I reviewed the changes and they look really good! :-)

I have two questions or potential changes before sending it to Brian, please check them below and let me know.

Thanks.

File: modules/apps/portal/portal-upgrade-impl/src/main/java/com/liferay/portal/upgrade/internal/report/UpgradeReport.java (hunk 1 above)
Comment (SamZiemer) [reply modules/apps/portal/portal-upgrade-impl/src/main/java/com/liferay/portal/upgrade/internal/report/UpgradeReport.java:176]: I just tested using this option, and it works just as well as my previous option. Regarding the performance, each was similar on my machine (around 1-2 seconds), however using FileUtils.sizeOfDirectory() is much cleaner. I will add that change in a new commit on top the current commit.

File: modules/apps/portal/portal-upgrade-impl/src/main/java/com/liferay/portal/upgrade/internal/report/UpgradeReport.java (hunk 2 above)
Comment (SamZiemer) [reply modules/apps/portal/portal-upgrade-impl/src/main/java/com/liferay/portal/upgrade/internal/report/UpgradeReport.java:172]: You are correct, appending /data/document_library is not needed and makes it point to the wrong directory (one that won't exist) I'll add this change on a commit and send to you.

**Comment by SamZiemer** [comment]
ci:test:sf

**Comment by SamZiemer** [comment]
ci:test:relevant

**Comment by SamZiemer** [comment]
ci:test:sf

File: modules/apps/portal/portal-upgrade-impl/src/main/java/com/liferay/portal/upgrade/internal/report/UpgradeReport.java (hunk 1 above)
Comment (achaparro) [reply modules/apps/portal/portal-upgrade-impl/src/main/java/com/liferay/portal/upgrade/internal/report/UpgradeReport.java]: perfect, thanks!

**Comment by SamZiemer** [comment]
ci:test:relevant

**Comment by achaparro** [comment]
see https://github.com/achaparro/liferay-portal/pull/1004

**Pull request closed without merging**